src/test/unit_tests/test_nc_with_a_couple_of_forks.js

```diff
@@ -0,0 +1,60 @@
+/* Copyright (C) 2024 NooBaa */
+'use strict';
+
+const path = require('path');
+const P = require('../../util/promise');
+const mocha = require('mocha');
+const assert = require('assert');
+const fs_utils = require('../../util/fs_utils');
+const { TMP_PATH, generate_nsfs_account, get_new_buckets_path_by_test_env, generate_s3_client, get_coretest_path } = require('../system_tests/test_utils');
+
+const coretest_path = get_coretest_path();
+const coretest = require(coretest_path);
+coretest.setup({ forks: 2, debug: 5 });
```

Comment:
this won't have any effect after coretest was already set up
why not setting forks=2 in the beforeAll and set it back to forks=1 instead of the new nc_index2.js and the makefile command?

Reply:
I created the new files because I want to make sure the coretest_nc was not up, and only this test sets it with a number of forks.
I'm not sure I understand - by changing the number of forks we need to restart the nsfs server, the number of forks will not be changed just by setting the env or changing the config file.

Reply:
From what I tried by trying to restart the `nsfs_process` (kill it and rerun it) the whole nc_core test is finishing.

Reply:
we discussed it, and I changed the code.